rl4co/envs/common/base.py

```diff
@@ -173,6 +174,12 @@ def get_action_mask(self, td: TensorDict) -> torch.Tensor:
         """
         raise NotImplementedError
 
+    def get_num_starts(self, td):
+        return get_num_starts(td, self.name)
```

Comment:
Awesome! 

At the moment this is not yet fully modularized (I guess each env should implement this function)
- In the decoding, i.e. [here](https://github.com/ai4co/rl4co/blob/66622a2e7d616e72696904e1552cf7d61dad817f/rl4co/models/nn/dec_strategies.py#L70) and [here](https://github.com/ai4co/rl4co/blob/66622a2e7d616e72696904e1552cf7d61dad817f/rl4co/models/nn/dec_strategies.py#L82) I guess we should eventually call instead `env.get_num_starts` and `env.select_start_nodes` instead of having to go through the default function from `ops` (i guess this is what you meant in the  discussion!)
- [Minor] we should also add some docstring
CC: https://github.com/ai4co/rl4co/discussions/117